assets/js/base/components/product-price/index.js

```diff
@@ -148,9 +147,7 @@ const ProductPrice = ( {
 				regularPriceStyle={ regularPriceStyle }
 			/>
 		);
-	}
-
-	if ( minPrice !== null && maxPrice !== null ) {
+	} else if ( minPrice !== null && maxPrice !== null ) {
```

Comment:
This price handles ranges. Did you test this fix on a discounted product with variations? I will do so now..

Reply:
In the previous versions of this code, we would return on each condition, so this refactor should be the same as the original behaviour.

Reply:
Seems to be working with variables. Doesn't appear that individual variations in the cart ever have ranges. Trying to figure out when this case is used.